@xen-orchestra/web-core/lib/components/icon/ObjectIcon.vue

```diff
@@ -1,3 +1,4 @@
+<!-- v1.0 -->
```

Comment:
Technically, ObjectIcon does not exist on Design System.

There is a Host icon, Network icon, Pool icon etc.

Each one is versioned independently.

Then, I suggest to add a version number for each one, instead of one general version for `ObjectIcon`

```suggestion
<!-- VM v1.0 -->
<!-- Host v1.0 -->
<!-- SR v1.0 -->
<!-- Backup Repository v1.0 -->
<!-- Network v1.0 -->
```